erpnext/patches/v12_0/update_item_tax_template_company.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+    item_tax_template_list = frappe.get_list('Item Tax Template')
```

Comment:
Reload the 'Item Tax Template` doc before running the patch, companies are not getting saved properly in all the templates on running the patch first time, I had to rerun it again manually. Rest all seems fine

Reply:
Ok thanks.